esm/interface/node.js

```diff
@@ -141,7 +141,7 @@ export class Node extends EventTarget {
     return false;
   }
 
-  getParent() {
+  _getParent() {
```

Comment:
Last question, sorry ... could we flag this somehow as `@private` ? I am not sure I want `_` prefixed methods in the TS definition file, although I don't use/know TS much, so if it's pointless, feel free to ignore.

Reply:
I'm not sure about TS but happy to add that 👍

Reply:
Is it private or protected? `EventTarget.dispatchEvent` is referring to `EventTarget. _getParent` but we need Node to overwrite `_getParent` and I'm a bit confused as how TS will behave in that case.

After checking TS docs and the net a bit I found this:
https://stackoverflow.com/questions/64083601/overriding-private-methods-in-typescript
which seem to confirm the use of `protected` if that's ok for you?

Reply:
whatever works, hoping it won't end up in the TS definition 👍